src/librustc/ty/mod.rs

```diff
@@ -2599,3 +2599,10 @@ impl fmt::Display for SymbolName {
         fmt::Display::fmt(&self.name, fmt)
     }
 }
+
+pub struct BorrowCheckResult {
+    /// contains the node-ids for variables within this function where the `mut`
+    /// declaration was used in some way (e.g., by modifying the variable's value,
+    /// or taking an `&mut` borrow of it).
+    used_mut_nodes: NodeSet
```

Comment:
huh, that's weird. It's used both in `check_loans.rs` and `gather_loans.rs`, I can't figure why that's cropping up.